### ISSUE TITLE
Trying to run this without DataParallel

### ISSUE THREAD

**Issue by rbunn80110** [rdcolema/pytorch-image-classification#2]
Removing the DataParallel from this snippet produces an error:

if arch.startswith('alexnet') or arch.startswith('vgg'):
            model.features = torch.nn.DataParallel(model.features)
            model.cuda()
        else:
            model = torch.nn.DataParallel(model).cuda()

AttributeError: 'ResNet' object has no attribute 'module'

--> 104         optimizer = optim.Adam(model.module.fc.parameters(), lr, weight_decay=1e-4)

Basically, I have a Titan X and two far inferior cards in my machine.  Running in DataParallel mode seriously impacts performance since the Titan X isn't used to its full capacity.  How would I change the code so that only the primary card runs?
